packages/client/src/components/formConfig/formTools/components.tsx

```diff
@@ -20,6 +20,19 @@ import {
 } from '@client/forms/configuration/formConfig/utils'
 import { Tooltip } from '@opencrvs/components/lib/icons'
 import ReactTooltip from 'react-tooltip'
+import { Event } from '@client/utils/gateway'
+import { BirthSection, DeathSection, ISerializedForm } from '@client/forms'
+import { useDefaultForm } from '@client/views/SysAdmin/Config/Forms/hooks'
+import { formSectionToFieldIdentifiers } from '@client/forms/questionConfig/transformers'
+import { useParams } from 'react-router'
+import { getFieldId } from '@client/forms/configuration/defaultUtils'
+import { IConditionalConfig } from '@client/forms/questionConfig'
+
+export const TitleWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+`
```

Comment:
This also can be replaced by `<Stack />`

Reply:
FYI <Text /> can take an icon. 
https://ui-kit.farajaland-staging.opencrvs.org/?path=/docs/typography-text--default

Reply:
Yes. Thanks for point it out.